### DISCUSSION TITLE
Dialog Box Loading in at Incorrect Position

### DISCUSSION THREAD

**Discussion by ValerieVasquez23** [dialogic-godot/dialogic#1391] [Q&A]
Hi! I'm using v1.4.5.
When I start a dialog, the dialog box should start in the bottom left corner, but instead it loads in at the right corner and then snaps to the left. How can I prevent this from happening or have the dialog box remain invisible until it is in the right position?

**Comment by coppolaemilio** [comment]
Sorry for the delay replying! I'm not sure what could be wrong in your current setup... Can you share a bit more how you start Dialogic in-game?
It could be an issue related to the current canvas layer or something like that, but I would need more information to figure out why it is happening 😅
